src/v/raft/tron/server.cc

```diff
@@ -87,8 +87,13 @@ class simple_group_manager {
       , _consensus_client_protocol(
           raft::make_rpc_client_protocol(self, clients))
       , _storage(
-          storage::kvstore_config(
-            1_MiB, 10ms, directory, storage::debug_sanitize_files::yes),
+          [directory]() {
+              return storage::kvstore_config(
+                1_MiB,
+                config::binding(10ms),
```

Comment:
I think I just noticed the `.bind()` usage below, but up until then I was confused about `config::binding(10ms)` and similar instance. Is that like defining a binding that isn't wired up and has a static value?

Reply:
Right.  This was too subtle, I'm changing it to add an explicit mock_binding helper function and make the special binding(T&&) constructor protected to make it less confusing.